README.md

```diff
@@ -1,10 +1,29 @@
-# tictactoe
+# TIC-TAC-TOE-GAME
 
-A new Flutter project.
+A Flutter project.
 
-## Getting Started
+## OBJECTIVE:
+This project for Data Day Grind 3.0:  with a aims to develop a Tic Tac Toe game using Flutter. It mainly consists of developing and implementing a computer program that plays Tic Tac Toe against another player.
```

Comment:
The computer doesn't play with the human, change it to "you can play this game with your friends"

Reply:
@ayushishu notify me for re-review once you've updated it